diff --git a/node_modules/@types/leaflet/index.d.ts b/node_modules/@types/leaflet/index.d.ts
index b6eb7e5..af5a416 100644
--- a/node_modules/@types/leaflet/index.d.ts
+++ b/node_modules/@types/leaflet/index.d.ts
@@ -368,6 +368,10 @@ export interface LeafletEventHandlerFnMap {
 
     tileerror?: TileErrorEventHandlerFn | undefined;
 
+    boxselectstart?: (() => void) | undefined;
+    boxselectmove?: ((event: { bounds: LatLngBounds, inverse: boolean }) => void) | undefined;
+    boxselectend?: ((event: { bounds: LatLngBounds, inverse: boolean }) => void) | undefined;
+
     // [name: string]: any;
     // You are able add additional properties, but it makes this interface uncheckable.
 }
@@ -2489,6 +2493,13 @@ export interface MapOptions {
     tapTolerance?: number | undefined;
     touchZoom?: Zoom | undefined;
     bounceAtZoomLimits?: boolean | undefined;
+
+    // SmoothWheelZoom
+    smoothWheelZoom?: boolean
+    smoothSensitivity?: number
+    
+    // MarkerSelect
+    boxSelect?: boolean
 }
 
 export type ControlPosition = "topleft" | "topright" | "bottomleft" | "bottomright";
